modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -233,6 +233,31 @@ const questionnaireStyle = theme => ({
         },
       },
     },
+    timelinePaper: {
+        padding: "8px 16px",
+        marginBottom: "40px"
+    },
+    timelineAncestor: {
+        opacity: 0.3
```

Comment:
We should probably bring them to full opacity on hover:
```suggestion
        opacity: 0.3,
        "&:hover": {
          opacity: 1,
        },
```